src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  background: #fff;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.1);
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__link {
+    display: flex;
+  }
+
+  &-nav {
+    display: flex;
+    height: 60px;
+    align-items: center;
+
+    &__list {
+      display: flex;
+      list-style: none;
+      font-family: Roboto, sans-serif;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should avoid specifying generic font-family names like 'sans-serif' as a fallback. You should include at least one more specific family as a fallback before the generic name, for example: 'Helvetica Neue', Arial, sans-serif.